### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by killerjn**
please updated readme

### CODE REVIEW

**Comment by mirh** [comment]
If just it was as easy as a simple number with GL /s

**Comment by killerjn** [comment]
I do not understand ?

**Comment by lightningterror** [comment]
4.0 Software, 4.3 Hardware.

**Comment by killerjn** [comment]
**I thought that the minimum required was opengl 3.3
according to this warning**
screenshot_20180519_141021

**Comment by lightningterror** [comment]
Check https://github.com/PCSX2/pcsx2/blob/master/plugins/GSdx/GLLoader.cpp

there you will find which extensions are required/mandatory.

**Comment by TheLastRar** [comment]
PCSX2 requires a number of extensions that aren't core to GL 3.3 (A driver can support it, but dosn't have to for compliance)
You can poke though GLLoader.cpp to see what extensions GSDX uses (and if they are mandatory).

If you where to recommend a GL version based on only core support, that that would be GL4.5 (GL_ARB_clip_control, GL_ARB_get_texture_sub_image & GL_ARB_texture_barrier)

**Comment by mirh** [comment]
The thing is, some extensions might have been mandatorily included in some newer version, but for as much it, they could be supported by OpenGL 2 cards (anisotropic landing officially in 4.6 is an example)

So, I mean.. Maybe just mentioning the direct3d feature level \*isn't that bad idea\*

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Development builds are also available from [our website](http://pcsx2.net/downlo
 ## Minimum
 * OS: Windows Vista SP2 or newer or GNU/Linux (32-bit or 64-bit)
 * CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up)
-* GPU: DirectX 10 GPU or better
+* GPU: DirectX 10 GPU or better (on linux Required Opengl 3.3)
```
Comment (baka0815) [inline README.md]: Its "Linux" with an uppercase "L", lowercase "r" in "required" and "OpenGL".

You might just rephrase it as "(or OpenGL 3.3 or better on Linux)"

Comment (mirh) [reply README.md]: "I'd just like to interject for a moment...."

Comment (baka0815) [reply README.md]: Sorry, I didn't follow the conversation. My bad.

**Comment by lightningterror** [comment]
`GPU: DirectX 10 GPU or better`

Was enough imo.

**Comment by mirh** [comment]
Isn't minimum D3D9 on Windows?

**Comment by lightningterror** [comment]
D3D10 capable GPUs are able to run GL if the driver supports it, at least I think.

**Comment by lightningterror** [comment]
I'll close this. There's no need to update the readme anyway at this time and the info provided is already there.
It will be updated when v1.6 is released.

**Pull request closed without merging**